specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/examples/AnalysisRun_Create.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "AnalysisRunPut",
+  "parameters": {
+    "api-version": "2022-06-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkVerifierName": "testNetworkVerifier",
+    "analysisIntentName": "testAnalysisIntent",
+    "analysisRunName": "testAnalysisRun",
+    "parameters": {
+      "name": "testAnalysisRun",
```

Comment:
I don't believe this is how the example should be? The user doesn't provide the name, type, or id explicitly. Name and id can be inferred from the uri, and type is determined on backend